### PR TITLE
improve install gateway doc

### PR DESCRIPTION
**Pull request by zirain**
Please provide a description for what this PR is for.

improve the doc, which may confuse user

fix: https://github.com/istio/istio/issues/38792

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/additional-setup/gateway/index.md [hunk 1]
```diff
@@ -205,6 +205,10 @@ your `Gateway` selector matches these labels.
 For example, in the above deployments, the `istio=ingressgateway` label is set on the gateway pods.
 To apply a `Gateway` to these deployments, you need to select the same label:
 
+{{< tabset category-name="gateway-selector" >}}
```
Comment (frankbu) [inline content/en/docs/setup/additional-setup/gateway/index.md:208]: I don't think adding these 3 tabs is making this any more clear.

I would suggest to instead change line 205 to say:
```
For example, in the above deployments, the `istio=istio-ingress` label is set on the gateway pods.
```
and change the two other (not helm) deployments to match:
```
     label:
        # Set a unique label for the gateway. This is required to ensure Gateways
        # can select this workload
        istio: istio-ingress
```
This way, you don't need to explain that the helm deployment is different in the example.

Comment (zirain) [reply content/en/docs/setup/additional-setup/gateway/index.md]: sounds good, updated.

**Review by frankbu — APPROVED** [review]

File: content/en/docs/setup/additional-setup/gateway/index.md (hunk 1 above)
Comment (frankbu) [reply content/en/docs/setup/additional-setup/gateway/index.md]: IIUC the helm install uses the deployment name for the label by default. If that's right, maybe a better idea is to leave the label as `istio: ingressgateway` and just change the helm command to use ingressgateway as the name. I'm not very familiar with helm, but it seems that the first argument for helm install is the name. If so, then I think that just changing the command to the following should work:
```
$ helm install ingressgateway istio/gateway -n istio-ingress
```
Then the gateway name will be the same in all 3 cases also. I wonder why it was different for helm in the first place.

Comment (frankbu) [reply content/en/docs/setup/additional-setup/gateway/index.md]: If you disagree, I've already approved, so you can remove the DNM label and merge as is.

Comment (zirain) [reply content/en/docs/setup/additional-setup/gateway/index.md]: I test with `helm install istio-ingressgateway istio/gateway -n istio-ingress`, which looks better

```
root@ecs-zirain:~# k get po -nistio-ingress
NAME                                   READY   STATUS    RESTARTS   AGE
istio-ingressgateway-685fcc78c-jltfj   1/1     Running   0          20s
root@ecs-zirain:~# k get po -nistio-ingress --show-labels
NAME                                   READY   STATUS    RESTARTS   AGE   LABELS
istio-ingressgateway-685fcc78c-jltfj   1/1     Running   0          28s   app=istio-ingressgateway,istio.io/rev=default,istio=ingressgateway,pod-template-hash=685fcc78c,service.istio.io/canonical-name=istio-ingressgateway,service.istio.io/canonical-revision=latest,sidecar.istio.io/inject=true

```

**Comment by zirain** [comment]
@frankbu I updated the implement, named all the gateway deployment with same name/labels.

if you agree, please remove `DNM`.

**Pull request merged**